src/main/java/individual/freshplace/config/auth/PrincipalDetailsService.java

```diff
@@ -0,0 +1,23 @@
+package individual.freshplace.config.auth;
+
+import individual.freshplace.entity.Member;
+import individual.freshplace.repository.MemberRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class PrincipalDetailsService implements UserDetailsService {
+
+    private final MemberRepository memberRepository;
+    private static UsernameNotFoundException exception = new UsernameNotFoundException("사용자를 찾을 수 없습니다.");
```

Comment:
```suggestion
    private final static UsernameNotFoundException exception = new UsernameNotFoundException("사용자를 찾을 수 없습니다.");
```
만약 상수로 쓴다면 `접근제어자 final static`은 붙여서 사용해주세요~